mrp_mto_with_stock/demo/product.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0"?>
+<!--
+Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+@author Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo> <data noupdate="1">
+
+        <record id="product_product_manufacture_1" model="product.product">
```

Comment:
could be the occasion to fix this 8 spaces indentation?

Reply:
I have notice same. Somewhere I have read that formatting changes not allow. So ignore it. May we allow to fix it in XML files ?

Reply:
Formatting changes are to avoid when you do a pull request for a change in an existing version. I'd say as we are doing a large diff anyway it could be the right moment to fix it. On the other hand, it can make the port of pull requests less easier. No strong opinion then.

Reply:
Corresponded.